components/apimgt/org.wso2.carbon.apimgt.rest.api/src/gen/java/org/wso2/carbon/apimgt/rest/api/TiersApi.java

```diff
@@ -42,5 +42,83 @@ public Response tiersGet(@ApiParam(value = "Media types acceptable for the respo
     throws NotFoundException {
     return delegate.tiersGet(accept,ifNoneMatch);
     }
+    @POST
+    
```

Comment:
Please note that these files get auto-generated using the swagger-to-cxf maven plugin we use for REST API development.